### PR TITLE
Add sDribbleToPoint

### PR DESCRIPTION
**Pull request by americast**
sDribbleToPoint.py has been added. All Python equivalents which were not discovered were originally parts of the commented section (of the c++ files), hence should not create an issue.

### CODE REVIEW

File: scripts/skills/sDribbleToPoint.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import skill_node
+import math
+
+def execute(param, state, bot_id):
+     point = Vector2D(param.DribbleToPointP.x, param.DribbleToPointP.y)
```
Comment (kvmanohar22) [inline scripts/skills/sDribbleToPoint.py]: Make sure arguments to `Vector2D()` are `ints`

File: scripts/skills/sDribbleToPoint.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+     ballPos = Vector2D(state.ballPos.x, state.ballPos.y)
+     ballSlope = ballPos.angle(botPos)
+     ballDist  = ballPos.dist(botPos)
+     ballTheta = math.fabs(ballDist.normalizeAngle(state.homePos[botID].theta - ballSlope));
+     #   if(ballDist > BOT_BALL_THRESH || ballTheta > DRIBBLER_BALL_ANGLE_RANGE):
+     #     return goToBall(param, state, botID) # No python equivalent found
+
+     obs=[]
+     o=obstacle()
+     for i in xrange (len(state.homeDetected)):
```
Comment (kvmanohar22) [inline scripts/skills/sDribbleToPoint.py]: Refer changes which are requested [here](https://github.com/krssg-ssl/skills_py/pull/15) on how to use `state.homeDetected`

File: scripts/skills/sDribbleToPoint.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	o.x = state.homePos[i].x
+	o.y = state.homePos[i].y
+	o.radius = 3 * BOT_RADIUS
+	obs.append(o)
+
+     for i in xrange(len(state.homeDetected)):
```
Comment (kvmanohar22) [inline scripts/skills/sDribbleToPoint.py]: Same as above for `state.homeDetected`

**Comment by americast** [comment]
@kvmanohar22 Incorporated changes requested.

File: scripts/skills/sDribbleToPoint.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+     botPos = Vector2D(int(state.homePos[botID].x), int(state.homePos[botID].y))
+     ballPos = Vector2D(int(state.ballPos.x), int(state.ballPos.y))
+     ballSlope = ballPos.angle(botPos)
+     ballDist  = ballPos.dist(botPos)
+     ballTheta = math.fabs(ballDist.normalizeAngle(state.homePos[botID].theta - ballSlope));
+     #   if(ballDist > BOT_BALL_THRESH || ballTheta > DRIBBLER_BALL_ANGLE_RANGE):
+     #     return goToBall(param, state, botID) # No python equivalent found
+
+     obs=[]; j=0
+     o=obstacle()
```
Comment (MayankB11) [inline scripts/skills/sDribbleToPoint.py]: The Obstacle class definiton is present in krssg-ssl/navigation_py/scripts/navigation/obstacle.py. You can ignore the unused parameters (x2, y2).

**Comment by americast** [comment]
@MayankB11 Fixed it. Thanx for pointing it out.

**Pull request merged**